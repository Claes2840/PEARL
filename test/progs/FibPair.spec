// Full evaluation
n = '4