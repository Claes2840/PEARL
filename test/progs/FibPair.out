() -> ()

init_1:
	entry
	exit
